src/main/java/fr/marcwrobel/jbanking/iban/RandomIban.java

```diff
@@ -0,0 +1,90 @@
+package fr.marcwrobel.jbanking.iban;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Random;
+
+/**
+ * Generates pseudorandom {@link Iban IBANs}.
+ *
+ * <p>
+ * Usage:
+ *
+ * <pre>
+ * // Generating a random IBAN
+ * Iban random1 = new RandomIban().next();
+ *
+ * // Generating a random IBAN using a given Random (in order to make the generation deterministic)
+ * Iban random2 = new RandomIban(new Random(0)).next();
+ *
+ * // Generating a random french or german IBAN
+ * Iban random3 = new RandomIban().next(BbanStructure.FR, BbanStructure.DE);
+ * </pre>
+ *
+ * <p>
+ * This class is not thread-safe and should be used only for tests. It is generating pseudorandom characters using
```

Comment:
Why isn't it thread safe? It doesn't have any state, so I don't think there should be any conflict?

Reply:
I thought `Random` was not thread-safe but I just looked at the documentation and I was wrong. Thanks !